db/upgrade.php

```diff
@@ -272,5 +272,17 @@ function xmldb_tool_excimer_upgrade($oldversion) {
         upgrade_plugin_savepoint(true, 2022011400, 'tool', 'excimer');
     }
 
+    if ($oldversion < 2022040801) {
+
+        // Change field 'parameters' into a text field.
+        $table = new xmldb_table('tool_excimer_profiles');
```

Comment:
would be safe to check that the field and db exists before doing anything. There is $dbman->table_exists() and $dbman->field_exists()

Reply:
done